packages/eds-tokens-build/src/index.ts

```diff
@@ -329,7 +332,7 @@ const _extend = ({
           },
         ],
       },
-      ts: {
+      /*       ts: {
```

Comment:
Why are we excluding the ts and JSON transforms?

Reply:
they were misnamed (-verbose) and we didn't have time to fix it